tests/appsec/iast/sink/test_hsts_missing_header.py

```diff
@@ -0,0 +1,59 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, released, missing_feature, weblog
+from .._test_iast_fixtures import SinkFixture
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
```

Comment:
Can you use manifest files please ? 

You also need to set a version number on java, otherwise, CI that tests old java tracer will break.

Reply:
fixed